_search-plugins/star-tree-index.md

```diff
@@ -0,0 +1,175 @@
+    - Changing Star Tree configuration will also require a re-index operation.
+- [Multi-values/array values]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/index/#arrays) are not supported
+- Only [limited queries and aggregations](#supported-query-and-aggregations) are supported with support for more coming in future
+- The cardinality of the dimensions should not be very high (like `_id` fields), otherwise it leads to storage explosion and higher query latencies.
+
+## Enabling star tree index
+- Set the feature flag `opensearch.experimental.feature.composite_index.star_tree.enabled"` to `true`. For more information about enabling and disabling feature flags, see [Enabling experimental features]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/experimental/).
+- Set the `indices.composite_index.star_tree.enabled` setting to `true`. For instructions on how to configure OpenSearch, see [configuring settings]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/index/#static-settings).
+- Set the `index.composite_index` index setting to `true` during index creation.
+
+## Examples
+
+The following examples show how to use star-tree index.
+
+### Defining star tree index in mappings
+
+Define star-tree configuration in index mappings when creating an index. <br/>
+To create star-tree index to pre-compute aggregations for `request_size` and `latency` fields for all the combinations of values in `port` and `status` fields indexed in the `logs` index, configure the following mapping:
+
+```json
+PUT logs
+{
+  "settings": {
+    "index.number_of_shards": 1,
+    "index.number_of_replicas": 0,
+    "index.composite_index": true
+  },
+  "mappings": {
+    "composite": {
+      "startree1": {
+        "type": "star_tree",
+        "config": {
+          "ordered_dimensions": [
+            {
+              "name": "status"
+            },
+            {
+              "name": "port"
+            }
+          ],
+          "metrics": [
+            {
+              "name": "size",
+              "stats": [
+                "sum"
+              ],
+              "name": "latency",
+              "stats": [
+                "avg"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "properties": {
+      "status": {
+        "type": "integer"
+      },
+      "port": {
+        "type": "integer"
+      },
+      "size": {
+        "type": "integer"
+      },
+      "latency": {
+        "type": "scaled_float",
+        "scaling_factor": 10
+      }
+    }
+  }
+}
+```
+
+For detailed information about Star Tree index mapping and parameters see [Star Tree field type]({{site.url}}{{site.baseurl}}/field-types/star-tree/).
+
+## Supported query and aggregations
+
+Star Tree index can be used to optimize aggregations for selected set of queries with support for more coming in upcoming releases.
```

Comment:
can we list down what is supported now (with 2.18)?

Reply:
We have listed down both queries and aggregations supported already - do you mean anything else ?

Reply:
Makes sense. 

May be it can be changed to "Star Tree index can be used to optimize aggregations. The list of supported queries is given below, support for more queries will be added in upcoming releases."